Communitheme/gtk-3.0/_common.scss

```diff
@@ -4550,12 +4550,25 @@ button.titlebutton {
   min-width: 24px;
   padding: 2px;
 
-  &:not(.close) {
+  &:not(.close):not(.appmenu){
     &:hover {
       @include draw_circle($c) ;
       &:backdrop:hover{@include draw_circle(darken($headerbar_fg_color,50%), $close:true)}
     }
   }
+  &.appmenu{
```

Comment:
I think you can replace this adding "button.titlebutton.appmenu" at row 1586

    - button, button.flat
    + button, button.flat, button.titlebutton.appmenu

Reply:
Ty